sdk/py/apepay/__init__.py

```diff
@@ -248,23 +248,24 @@ def streams_by_creator(self, creator: AddressType) -> Iterator["Stream"]:
         for stream_id in range(self.contract.num_streams(creator)):
             yield Stream(manager=self, creator=creator, stream_id=stream_id)
 
-    def all_streams(self) -> Iterator["Stream"]:
+    def all_streams(self, start_block: Optional[int] = None) -> Iterator["Stream"]:
         for stream_created_event in self.contract.StreamCreated.range(
-            self.contract.receipt.block_number, self.chain_manager.blocks.head.number
+            start_block or self.contract.receipt.block_number,
+            self.chain_manager.blocks.head.number,
         ):
             yield Stream.from_event(
                 manager=self,
                 event=stream_created_event,
                 is_creation_event=True,
             )
 
-    def active_streams(self) -> Iterator["Stream"]:
-        for stream in self.all_streams():
+    def active_streams(self, start_block: Optional[int] = None) -> Iterator["Stream"]:
+        for stream in self.all_streams(start_block):
```

Comment:
```suggestion
        for stream in self.all_streams(start_block=start_block):
```

since it an optional